tabnames.py

```diff
@@ -0,0 +1,31 @@
+import org.bukkit as bukkit
+from helpers import *
+from org.bukkit import *
+
+tabnames_version = "v1.0.0"
+
+##############################################################
+#                                                            #
+# This module automatically puts people in the corresponding #
+# scoreboard team so that their name is colored properly and #
+# tab will be nicely sorted.                                 #
+#                                                            #
+##############################################################
+
+ranks = ["visitor", "member", "builder", "trusted", "modintraining", "mod", "admin", "breaker"]
+prefixes = {"admin":"a", "mod":"b", "modintraining":"c", "trusted":"d", "builder":"e", "member":"f","visitor":"g"}
+
+@hook.event("player.PlayerJoinEvent", "low")
+def on_player_join(event):
+    scoreboard_team = prefix(get_Rank(event.getPlayer()))
+    bukkit.Bukkit.getServer().dispatchCommand(bukkit.Bukkit.getServer().getConsoleSender(),
+        "scoreboard teams join " + scoreboard_team + " " + event.getPlayer().getName())
+
+def get_Rank(player):
+    for i in range(0, len(ranks) - 1):

```

Comment:
If I did it like that it would return the first rank the user does not have anymore. Which is not quite the rank I need. I could go from the highest to the lowest rank and find which one the user has as his first rank but dico told me to do it this way